config/initializers/elasticsearch.rb

```diff
@@ -1,3 +1,5 @@
 # NOTE: the `search.rb` initializer relies on this file being loaded first,
 # which it currently manages because it comes first in the alphabet.
-GovukNeedApi.search_client = Elasticsearch::Client.new(host: "localhost:9200")
+GovukNeedApi.search_client = Elasticsearch::Client.new(
+  hosts: ENV['ELASTICSEARCH_HOSTS'].split(',')
+)
```

Comment:
It looks like I forgot to set the environment variables on the dev VM, sorry.

Similar to the YAML change in https://github.com/alphagov/govuk-puppet/commit/854b2a7c179f588b864700e6731b22671e56acd3, you need to add these lines alphabetically to `hieradata/development.yaml`:

```
govuk::apps::need_api::elasticsearch_hosts: 'localhost:9200'
govuk::apps::need_api::redis_host: '127.0.0.1'
```